src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -167,3 +167,33 @@ const applicantRelationshipEventMap = {
 export function recordApplicantRelationship({ formData: { onBehalfOf } }) {
   return <UserInteractionRecorder eventRecorder={recordEvent} selectedValue={onBehalfOf} trackingEventMap={applicantRelationshipEventMap}></UserInteractionRecorder>;
 }
+
+/**
+ * Provides an issue example tooltip with the provided text
+ * @param {string} Example text
+ */
+export const getIssueLabel = (labelText, exampleText) => {
```

Comment:
👏 Nicely done. I do wonder if there's more semantic markup to use here, but nothing comes to mind.

Reply:
Probably needs a tab index if we want screen readers to see these similar to the suggestion in [this](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/12670). defer to @1Copenut on this

Reply:
Good point, @erikphansen, updated based on @1Copenut's [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12252#issuecomment-419987168) regarding how this can be structured.